movies/1925/the-wizard-of-oz.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name" : "The Wizard of Oz",
```

Comment:
This file needs formatting, all colons should follow immediately after the key.   e.g "name": "The Wiz..."